client/dashboard/data/purchase.ts

```diff
@@ -65,18 +63,6 @@ export interface PurchasePriceTier {
 	maximumPriceDisplay?: string | null;
 }
 
-export interface RawPurchasePriceTierEntry extends PriceTierEntry {
-	/**
-	 * The formatted minimum price for the tier.
-	 */
-	minimum_price_monthly_display: string;
```

Comment:
These properties are returned as part of the tier data from the products endpoints, not the purchases endpoints.